Client/mods/deathmatch/logic/CClientEntity.h

```diff
@@ -198,6 +198,7 @@ class CClientEntity : public CClientEntityBase
 
     CCustomData*  GetCustomDataPointer() { return m_pCustomData; }
     CLuaArgument* GetCustomData(const char* szName, bool bInheritData, bool* pbIsSynced = nullptr);
+    CLuaArguments* GetAllCustomData(CLuaArguments* table);
     bool          GetCustomDataString(const char* szKey, SString& strOut, bool bInheritData);
     bool          GetCustomDataFloat(const char* szKey, float& fOut, bool bInheritData);
```

Comment:
Please format the document! (I can tell this is not formatted correctly because the types will all line up.)

Reply:
Yeah sure